<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>700</width>
      <height>500</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>评分</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_import">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>100</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>导入遗器数据</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_setname">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>10</y>
        <width>120</width>
        <height>24</height>
       </rect>
      </property>
      <property name="currentText">
       <string notr="true">不限</string>
      </property>
      <item>
       <property name="text">
        <string>不限</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="comboBox_position">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>10</y>
        <width>120</width>
        <height>24</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>不限</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>头</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>手</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>衣服</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>鞋</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>球</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>绳</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="comboBox_config">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>10</y>
        <width>120</width>
        <height>24</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>银狼</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>饮月</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>黑塔</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>布洛妮娅</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>停云</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>镜流</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>阮梅</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>符玄</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pushButton_calscore">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>10</y>
        <width>100</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>计算</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>650</width>
        <height>375</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>潜力</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
